Makefile

```diff
@@ -0,0 +1,144 @@
+###########################################################################
+#
+#  Copyright (c) 2013-2015, ARM Limited, All Rights Reserved
+#  SPDX-License-Identifier: Apache-2.0
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may
+#  not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+###########################################################################
+# App
+APP:=$(notdir $(shell 'pwd'))
+
+# Toolchain
+PREFIX:=arm-none-eabi-
+GDB:=$(PREFIX)gdb
+OBJDUMP:=$(PREFIX)objdump
+
+MBED_TARGET:=K64F
```

Comment:
I suggest you remove this altogether. The user always needs to specify the MBED_TARGET. It helps avoiding confusion (typical scenario: I do `make MBED_TARGET=DISCO_F429ZI debug gdb`, then change something, then do `make debug gdb` (I forgot the target): K64F gets compiled and I end up debugging the old DISCO firmware.